scripts/e2e_test/copy_artifacts_to_server.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -x
+
+source venv/bin/activate
+swcli instance login http://$1 --username starwhale --password abcd1234 --alias pre-k8s
```

Comment:
use other alias name?

Reply:
modified to `source $WORKDIR/source venv/bin/activate`